deploy_stack.ps1

```diff
@@ -0,0 +1,11 @@
+#!ps1
+
+if (Get-Command docker -errorAction SilentlyContinue)
+{
+    Write-Host "Deploying stack"
+    docker stack deploy func --compose-file ./docker-compose.yml
+}
+else

```

Comment:
If adding error handling for Windows - should that also be done in the Linux versions?